test/conformance/CMakeLists.txt

```
@@ -143,18 +156,20 @@ if(UR_DPCXX)
         string(REPLACE "," ";" TARGET_TRIPLES ${UR_CONFORMANCE_TARGET_TRIPLES})
     endif()
 
-    add_subdirectory(device_code)
-    add_subdirectory(kernel)
-    add_subdirectory(program)
-    add_subdirectory(enqueue)
-    add_subdirectory(integration)
-    add_subdirectory(exp_command_buffer)
-    add_subdirectory(exp_enqueue_native)
-    add_subdirectory(exp_usm_p2p)
-    add_subdirectory(exp_launch_properties)
-    add_subdirectory(memory-migrate)
+    foreach(dir ${TEST_SUBDIRECTORIES_DPCXX})
+        add_subdirectory(${dir})
+    endforeach()
 else()
+    set(DISABLED_TESTS)
```

NIT: I think it is better to set this as empty string
```suggestion
    set(DISABLED_TESTS "")
```